resolwe_bio/processes/workflows/quantseq.py

```diff
@@ -0,0 +1,369 @@
+"""3' mRNA-Seq pipeline comprised of QC, preprocessing, alignment and quantification."""
+
+from resolwe.process import (
+    BooleanField,
+    Data,
+    DataField,
+    FloatField,
+    GroupField,
+    IntegerField,
+    ListField,
+    Process,
+    StringField,
+)
+from resolwe.process.models import Process as BioProcess
+
+
+class WorkflowQuantSeq(Process):
+    """3' mRNA-Seq pipeline.
+
+    Reads are preprocessed by __BBDuk__ or __Cutadapt__ which removes adapters,
+    trims reads for quality from the 3'-end, and discards reads that are too
+    short after trimming. Preprocessed reads are aligned by __STAR__
+    aligner. For read-count quantification, the __FeatureCounts__ tool
+    is used. QoRTs QC and Samtools idxstats tools are used to report
+    alignment QC metrics.
+
+    QC steps include downsampling, QoRTs QC analysis and alignment of
+    input reads to the rRNA/globin reference sequences. The reported
+    alignment rate is used to assess the rRNA/globin sequence depletion
+    rate.
+    """
+
+    slug = "workflow-quantseq"
+    name = "QuantSeq workflow"
+    requirements = {
+        "expression-engine": "jinja",
+    }
+    data_name = "{{ reads|sample_name|default('?') }}"
+    version = "5.0.0"
+    process_type = "data:workflow:quant:featurecounts"
+    category = "Pipeline"
+
+    class Input:
+        """Input fields."""
+
+        trimming_tool = StringField(
+            label="Trimming tool",
+            choices=[
+                ("bbduk", "BBDuk"),
+                ("cutadapt", "Cutadapt"),
+            ],
+            description="Pick the trimming tool.",
```

Comment:
Expand the description. Explain that BBDuk requires an additional input file (adapters). Explain that the Cutadapt tool includes pre-determined adapters sequences to be removed.